jenkins/scripts/files/run_integration_tests.sh

```diff
@@ -104,6 +107,10 @@ then
 
 elif [[ "${REPO_NAME}" == "project-infra" ]]
 then
+  # FIXME(hardys) only for testing
+  # Build/test the sushy-tools/vbmc images, since they are defined in this repo
+  export SUSHY_TOOLS_LOCAL_IMAGE="/home/${USER}/tested_repo/resources/sushy-tools"
+  export VBMC_LOCAL_IMAGE="/home/${USER}/tested_repo/resources/vbmc"
```

Comment:
@hardys Can you please try with these changes, because it seems we are not giving the right path here. In this scenario, Metal3-dev-env is cloned to [`metal3`](https://github.com/metal3-io/project-infra/blob/c3e4635fef344c839516b068abd52a3dc0d78e2b/jenkins/scripts/files/run_integration_tests.sh#L126)
```suggestion
  export SUSHY_TOOLS_LOCAL_IMAGE="metal3/resources/sushy-tools"
  export VBMC_LOCAL_IMAGE="metal3/resources/vbmc"
```